doc/classes/ParticleProcessMaterial.xml

```diff
@@ -193,7 +194,7 @@
 		</member>
 		<member name="emission_curve" type="Texture2D" setter="set_emission_curve" getter="get_emission_curve">
 			Each particle's color will be multiplied by this [CurveTexture] over its lifetime.
-			[b]Note:[/b] This property won't have a visible effect unless the render material is marked as unshaded.
+			[b]Note:[/b] [member emission_curve] multiplies the particle mesh's vertex colors. To have a visible effect on a [BaseMaterial3D], [member BaseMaterial3D.vertex_color_use_as_albedo] [i]must[/i] be [code]true[/code]. For a [ShaderMaterial], [code]ALBEDO *= COLOR.rgb;[/code] must be inserted in the shader's [code]fragment()[/code] function. Otherwise, [member color] will have no visible effect.
```

Comment:
```suggestion
			[b]Note:[/b] [member emission_curve] multiplies the particle mesh's vertex colors. To have a visible effect on a [BaseMaterial3D], [member BaseMaterial3D.vertex_color_use_as_albedo] [i]must[/i] be [code]true[/code]. For a [ShaderMaterial], [code]ALBEDO *= COLOR.rgb;[/code] must be inserted in the shader's [code]fragment()[/code] function. Otherwise, [member emission_curve] will have no visible effect.
```